lib/promscrape/discovery/openstack/openstack.go

```diff
@@ -25,10 +25,10 @@ type SDConfig struct {
 	Region                      string `yaml:"region"`
```

Comment:
All the fields except of `role` and `region` are optional here

Reply:
all fixed